examples/nodejs/observability.ts

```diff
@@ -0,0 +1,55 @@
+import setupObservability from './utils/instrumentation';
+setupObservability('observability-example');
```

Comment:
I might add a comment explaining what this line is doing behind the scenes.

Also in a subsequent PR I think we might want to consider splitting these out to two separate examples since the instrumentation logic is so different. When they are combined I wonder if users won't understand which parts are providing metrics vs. traces.